definitions/tools/optitype_dna.cwl

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: 'Run optitype on dna samples'
+baseCommand: ["/bin/bash", "/usr/bin/optitype_script.sh"]
+arguments:
+    [ { valueFrom: $(runtime.outdir) },
```

Comment:
Should this one point to the `runtime.tmpdir` instead?

Reply:
Agreed, fixed in commit bf327ad